report/components/core/Labels.tsx

```diff
@@ -84,6 +84,9 @@ const _AuthorLabel = ({ index }: LabelProps) => {
     } else if (platform === "telegram") {
         // TODO: two letters
         let letter: string = "";
+        if (author.n === undefined) {
+            throw new Error("Telegram author name undefined (only Discord name's can be undefined)");
```

Comment:
This is very dangerous, throwing an error in something we don't know for sure it is true may crash the app unnecessarily.

Reply:
Before, author.n couldn't be undefined at all, so wouldn't an error have been thrown in this case anyways?